modules/oe_whitelabel_helper/config/install/daterange_compact.format.oe_whitelabel_date_time_long.yml

```
@@ -0,0 +1,16 @@
+langcode: en
+status: true
+dependencies: {  }
+id: oe_whitelabel_date_time_long
+label: 'Date and time, long names'
+default_pattern: 'l d F Y, H.i (T)'
```

I see no leading zero in the pdf.
> Tuesday 8 February 2022, 14.30 - Wednesday 9 February 2022, 16.30
```suggestion
default_pattern: 'l j F Y, H.i (T)'
```